src/main/webapp/scripts/tests/popular-test.js

```diff
@@ -0,0 +1,45 @@
+class MockVideoResponse {
+  constructor() {
+    this.result = 'result';
+  }
+}
+
+QUnit.test('list-video-service - getPopularRequest', function(assert) {
+  const regionCode = 'US';
+  const result = getPopularRequest(regionCode);
+  assert.equal(result.regionCode, regionCode, 'Expecting value US');
+});
```

Comment:
You want to test behavior instead of methods. You should be testing that when the main script loads, it loads properly:

(1) Properly takes an input and converts into an API request (in this case there is no user input so no need to check that the api request was properly built)

(2) Properly takes the response from the mocked API (which might do some kind of processing) and displays it correctly.

I wouldn't necessarily test each js file in this PR since they are treated like helper methods and there is not much processing in each individual file. For example, in list video service, you call the api and just return the response. No need to individually test that because there is no processing happening. If you took the response and returned processed data then it that case you would need to test that. We can talk more about testing in this PR in our chat.